framework/include/cppmicroservices/Constants.h

```diff
@@ -43,7 +43,7 @@ namespace Constants {
 
 /**
  * Location identifier of the OSGi <i>system bundle </i>, which is defined
- * to be &quot;System Bundle&quot;.
+ * to be &quot;System %Bundle&quot;.
  */
 US_Framework_EXPORT extern const std::string SYSTEM_BUNDLE_LOCATION; // = "System Bundle";
```

Comment:
I was thinking about this as well but thought I'd leave it like this for now. It is not something clients come in contact with directly.